packages/cactus-plugin-satp-hermes/src/main/typescript/core/stage-handlers/stage0-handler.ts

```diff
@@ -226,6 +226,7 @@ export class Stage0SATPHandler implements SATPHandler {
 
       return message;
     } catch (error) {
+      this.Log.debug(`Crash in ${fnTag}`, error);
```

Comment:
@LordKubaya Suggestion: I think it might be worthwhile to leave this in until the error handling is fixed. These log statements might be the only place these errors get logged with the current setup (if in the meantime it was fixed then please disregard this comment of mine).

Reply:
@LordKubaya yeah as @petermetz mentioned I left that in because the error handling isn't 100% working, but since you are refactoring that already, I can remove these changes if you prefer it (and since the image was built with them, these logs will still appear in case something goes wrong until a image is pushed).

Reply:
@petermetz Yes this is being worked on now. But I was saying to add this log to all the handlers for now.

Reply:
@LordKubaya Oh, facepalm. Sorry, I misread what you wrote. Then we are in agreement.